hs/server.hs

```diff
@@ -29,21 +32,20 @@ about = "about"
 static :: String
 static = "static"
 
-staticDir :: String
---staticDir = "C:\\Users\\David\\Documents\\courseography"
-staticDir = "/home/cynic/4/courseography"
-
 course :: String
 course = "course"
 
 main :: IO ()
-main = simpleHTTP nullConf $
-  msum [ dir grid $ gridResponse,
-         dir graph $ graphResponse,
-         dir about $ aboutResponse,
-         dir static $ serveDirectory EnableBrowsing [] staticDir,
-         dir course $ path (\s -> liftIO $ queryCourse s)
-       ]
+main = do
+    cwd <- getCurrentDirectory
+    let staticDir = encodeString $ parent $ fromText (T.pack cwd)
```

Comment:
I used encodeString because the result of `parent` is a `Filesystem.Path.CurrentOS.FilePath`, which is unrecognised by Happstack. It either accepts a `String` or a `Prelude.FilePath`. From what I tried, decodeString will not work.